support/dev-tools/polkadot-utils/src/lib/TxnBatcher.test.ts

```diff
@@ -0,0 +1,59 @@
+import { TxnBatcher } from './TxnBatcher';
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import { Keyring } from '@polkadot/keyring';
+import settings from '../settings';
+
+async function createPromiseApi(nodeAddress: string, types: any) {
+    const provider = new WsProvider(nodeAddress);
+    const api = await new ApiPromise({ provider, types });
+    await api.isReady;
+    return api;
```

Comment:
IIUC, this requires having a local node running for these tests to pass? That's not a good idea. Tests should start up, execute, and finish fully in isolation.